app/api/events.py

```diff
@@ -104,7 +105,7 @@ def validate_timezone(self, data, original_data):
     organizer_description = fields.Str(allow_none=True)
     is_sessions_speakers_enabled = fields.Bool(default=False)
     privacy = fields.Str(default="public")
-    state = fields.Str(default="Draft")
+    state = fields.Str(validate=validate.OneOf(choices=["published", "draft"]), allow_none=True, default='draft')
```

Comment:
Here, published and draft are not the only acceptable values. Please populate the above validator with all possible values.
While,
> Only of sessions with state approved or accepted

means is speaker with approved or accepted sessions should be there when a `GET` request is done. For this you'd have to modify the query method here I think